relint.py

```diff
@@ -69,8 +111,13 @@ def main():
 
     tests = list(load_config(args.config))
 
+    diff = None
+    if args.diff:
+        output = subprocess.run(['git', 'diff', '-U0'], stdout=subprocess.PIPE)
```

Comment:
```suggestion
        output = subprocess.check_output(['git', 'diff', '-U0', '--cached'])
```

Reply:
See also https://stackoverflow.com/questions/1587846/how-do-i-show-the-changes-which-have-been-staged